src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/appsettings.json

```diff
@@ -11,10 +11,11 @@
 //  },
 ////#elseif (OrganizationalAuth)
 //  "AzureAd": {
+  //    "Instance": "https:////login.microsoftonline.com/",
+  //    "Autority": "{Instance}{TenantId}/v2.0",
```

Comment:
Use the same order in both templates.

Reply:
Indentation too

Reply:
@Tratcher : the templates are in a different order already today (I was surprised too). I did not change the order.